### ISSUE TITLE
html-proofer warnings

### ISSUE THREAD

**Issue by gvarisco** [italia/developers.italia.it#69]
bundle exec htmlproofer ./_site  --check-html --allow-hash-href --empty-alt-ignore --only-4xx --disable-external introduced in #68 reports:

```
Ran on 30 files!


- ./_site/en/18app/index.html
  *  internally linking to /en/note-legali, which does not exist (line 818)
     <a href="/en/note-legali">Terms and conditions</a>
  *  internally linking to /en/privacy-policy, which does not exist (line 815)
     <a href="/en/privacy-policy">Privacy policy</a>
- ./_site/en/anpr/index.html
  *  internally linking to /en/note-legali, which does not exist (line 846)
     <a href="/en/note-legali">Terms and conditions</a>
  *  internally linking to /en/privacy-policy, which does not exist (line 843)
     <a href="/en/privacy-policy">Privacy policy</a>
- ./_site/en/cie/index.html
  *  internally linking to /en/note-legali, which does not exist (line 818)
     <a href="/en/note-legali">Terms and conditions</a>
  *  internally linking to /en/privacy-policy, which does not exist (line 815)
     <a href="/en/privacy-policy">Privacy policy</a>
- ./_site/en/credits/index.html
  *  internally linking to /en/note-legali, which does not exist (line 1618)
     <a href="/en/note-legali">Terms and conditions</a>
  *  internally linking to /en/privacy-policy, which does not exist (line 1615)
     <a href="/en/privacy-policy">Privacy policy</a>
- ./_site/en/datigov/index.html
  *  internally linking to /en/note-legali, which does not exist (line 857)
     <a href="/en/note-legali">Terms and conditions</a>
  *  internally linking to /en/privacy-policy, which does not exist (line 854)
     <a href="/en/privacy-policy">Privacy policy</a>
- ./_site/en/faq/index.html
  *  internally linking to /en/note-legali, which does not exist (line 885)
     <a href="/en/note-legali">Terms and conditions</a>
  *  internally linking to /en/privacy-policy, which does not exist (line 882)
     <a href="/en/privacy-policy">Privacy policy</a>
- ./_site/en/fatturapa/index.html
  *  internally linking to /en/note-legali, which does not exist (line 818)
     <a href="/en/note-legali">Terms and conditions</a>
  *  internally linking to /en/privacy-policy, which does not exist (line 815)
     <a href="/en/privacy-policy">Privacy policy</a>
- ./_site/en/index.html
  *  internally linking to /en/note-legali, which does not exist (line 1199)
     <a href="/en/note-legali">Terms and conditions</a>
  *  internally linking to /en/privacy-policy, which does not exist (line 1196)
     <a href="/en/privacy-policy">Privacy policy</a>
- ./_site/en/pagopa/index.html
  *  internally linking to /en/note-legali, which does not exist (line 818)
     <a href="/en/note-legali">Terms and conditions</a>
  *  internally linking to /en/privacy-policy, which does not exist (line 815)
     <a href="/en/privacy-policy">Privacy policy</a>
- ./_site/en/projects/index.html
  *  internally linking to /en/note-legali, which does not exist (line 1294)
     <a href="/en/note-legali">Terms and conditions</a>
  *  internally linking to /en/privacy-policy, which does not exist (line 1291)
     <a href="/en/privacy-policy">Privacy policy</a>
- ./_site/en/spid/index.html
  *  internally linking to /en/note-legali, which does not exist (line 872)
     <a href="/en/note-legali">Terms and conditions</a>
  *  internally linking to /en/privacy-policy, which does not exist (line 869)
     <a href="/en/privacy-policy">Privacy policy</a>
htmlproofer 3.7.2 | Error:  HTML-Proofer found 22 failures!
```

**Comment by gvarisco** [comment]
@rasky FYI - this is the reason why circleci builds are failing. /cc @pdavide

**Comment by pdavide** [comment]
Already fixed in #68 commit https://github.com/italia/developers.italia.it/commit/11f055b46563666bf463bfe1084d9fa717ec6962
